kubernetes/deploy-testing.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
```

Comment:
Do we want this file to be a .tmpl file instead of .yaml?

Reply:
absolutely, thanks